filenames.auto.gni

```diff
@@ -86,6 +86,7 @@ auto_filenames = {
     "docs/api/structures/notification-action.md",
     "docs/api/structures/point.md",
     "docs/api/structures/printer-info.md",
+    "docs/api/structures/process-memory-info.md",
```

Comment:
should have been updated in [docs: process.getProcessMemoryInfo() returns a Promise](https://github.com/electron/electron/pull/16593)